aptabase/build.gradle

```diff
@@ -42,4 +42,17 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.5'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
+}
+
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                from components.release
+                groupId = 'com.aptabase'
+                artifactId = 'Aptabase-Android'
```

Comment:
seems like the convention for artifactId is all lowercase?

Reply:
maybe just `aptabase` or `aptabase-kotlin`?

Reply:
Changed it to `aptabase-kotlin` to match the library name.